CHANGELOG.md

```diff
@@ -7,9 +7,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Added
 - Per profile Docker resources of WSO2 Identity Server and Analytics v5.9.x for Alpine, CentOS and Ubuntu
+- Environment variables indicating MySQL user credentials
```

Comment:
@Savidude better mention about adding new Docker Compose resources for IAM v5.9.0 rather than mentioning about individual improvements. Applies to the `Changed` section change, as well.

Reply:
Done https://github.com/wso2/docker-is/pull/170/commits/3a68e787cc3928eee17f002943e939f23eed387e